src/components/FeatureViews/Courses/ClassInfo.js

```diff
@@ -95,18 +95,21 @@ const UPDATE_COURSE_LINK = gql`
             courseLink: $courseLink
             courseLinkDescription: $courseDescription
             id: $id
-            availabilities: {
-                dayOfWeek: MONDAY
-                endTime: "10:00"
-                startTime: "12:00"
-            }
+            availabilities: [
+                { dayOfWeek: SATURDAY, startTime: "01:00", endTime: "09:00" }
+                { dayOfWeek: SUNDAY, startTime: "01:00", endTime: "09:00" }
```

Comment:
put the existing course availabilities in this array - this will cause all courses where the course link is edited to have these availabilities